packages/flutter_tools/lib/src/commands/clean.dart

```diff
@@ -36,6 +43,7 @@ class CleanCommand extends FlutterCommand {
 
   @override
   Future<FlutterCommandResult> runCommand() async {
+    print(argResults);
```

Comment:
Will remove

Reply:
I don't see `null` when I run this locally. Are you ensuring the cached tool snapshot is up to date? See https://github.com/flutter/flutter/wiki/The-flutter-tool#making-changes-to-the-flutter-tool for more context.

Reply:
will check that.

Currently i run my unit test doing that : 
>  dart bin/flutter_tools.dart test test/commands.shard/hermetic/clean_test.dart

Reply:
> > will check that.
> > Currently i run my unit test with this command :
> > ```
> > $>  dart bin/flutter_tools.dart test test/commands.shard/hermetic/clean_test.dart
> > ```
> 
> Well if it's showing null in unit tests, that's different :) That means your test is set up wrong.

Need help on that, didn't know what is wrong inside the test...

Reply:
I pushed a new version with a working test. Maybe not the best test ever, do not hesitate to tell what change need I do.